src/Microsoft.Health.Fhir.Shared.Api/Features/ActionResults/FhirResult.cs

```diff
@@ -68,6 +92,17 @@ public static FhirResult NotFound()
             };
         }
 
+        /// <summary>
+        /// Returns a BadRequest response
+        /// </summary>
+        public static FhirResult BadRequest()
```

Comment:
Is this used anywhere?

Reply:
No Will remove it.

Reply:
Is it only used in the tests? If so, then this should be in a test project.